Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM buildpack-deps:xenial
 
 ENV GO_VERSION 1.8
-ENV KUBECTL_VERSION 1.5.3
+ENV KUBECTL_VERSION 1.6.0
```

Comment:
Instead of replacing this value, you create consts for this and pass the value? This is needed because we want ACS Engine customers to be able to create 1.5.3 and 1.6.0 cluster. I will soon be working on adding orchestratorVersion field to orchestratorProfile to enable this.